library/IvozProvider/Model/ExternalCallFilters.php

```diff
@@ -29,31 +29,19 @@ public function init()
     }
 
     /**
-     * Check if the given number matches the regular expression
-     *
-     * @param string $origin in E164 form
-     * @return true if number matches, false otherwise
-     */
-    public function matchOrigin($origin, $regexp)
-    {
-        // Nothing matches empty expressions
-        if (is_null($regexp) || empty($regexp)) {
-            return false;
-        }
-
-        // Check if origin matches
-        return preg_match("/$regexp/", "$origin");
-    }
-
-
-    /**
      * Check if the given number matches External Filter black list
      * @param string $origin in E164 form
      * @return true if number matches, false otherwise
      */
     public function matchBlackList($origin)
```

Comment:
Seems good. I just kept the original name of the methods that existed some time ago, but we can revamp them now.